src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocStyleCheck.java

```diff
@@ -172,6 +214,54 @@
  *   &lt;property name="checkFirstSentence" value="false"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public class Test {
+ *     &#47;**
+ *      * Some description here // OK
+ *      * Another description // violation, the sentence must end with a proper punctuation
```

Comment:
`Another description` 
something line `second line of description` looks better